mysql/dbstat/dbstat.go

```diff
@@ -484,19 +473,27 @@ func (s *MysqlStatDBs) GetQueryResponseTime() {
 		return
 	}
 
-	for i, time := range res["time"] {
+	for i := 0; i < len(res["time"]); i++ {
+		res["time"][i] = strings.TrimSpace(res["time"][i])
```

Comment:
should have a loop-local variable and use that instead of re-assigning to `res[time][i]`